<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>835</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1180</width>
    <height>835</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Interface Pilote ( IHM )</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>560</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Température :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="OpenCV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Rounded MT Bold'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;Si Vous êtes Connecté ...&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:600;&quot;&gt;Cliquer sur Démarrer la capture Vidéo.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Mode_Manuelle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode Manuelle</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Connecter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_temperature">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_trameGPS">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_MaximumChargeMineraux">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Mode_Auto">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode Automatique</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Charge maximum de mineraux dans le Bac</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connecter avec le robot ? </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>380</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Localisation du robot (trame):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Connection">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>770</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Checkpoint">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>410</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Checkpoint</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>840</y>
     <width>1181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>0</y>
     <width>20</width>
     <height>851</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_trameGPS_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Localisation du robot (trame split):</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>651</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
    </font>
   </property>
   <property name="title">
    <string>Control Robot</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Réponse robot : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Direction_Avancer">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Avancer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Hauteur_Pince">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_instruction">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Vitesse">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Direction_Gauche">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Direction_Droite">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vitesse">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instruction Données : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vitesse :</string>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hauteur de la Pince :</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_Hauteur_Pince">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>31</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>33</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Direction_Reculer">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reculer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_instruction_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Reculer_Gauche">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reculer a gauche</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Reculer_Droite">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reculer à droite</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderFermeturePince">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>33</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fermeture de la pince :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Connecter_Instruction">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>810</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
text-align:center;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer_Color_black">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>670</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choississez une couleur pour l'interface</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer_Color_blue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>670</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer_Color_rouge">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer_Color_white">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>670</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Envoyer_Color_Base">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>670</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_geolocalisation">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_geolocalisation_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Geolocalisation état : </string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Local_Layout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="OpenCvFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Nom_Site">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>740</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DemarrageVideo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Démarrer la capture vidéo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_VoirNomSite">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>740</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Voir Nom et Site</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>740</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>S'identifier</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Requete_Robot">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>380</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_connecte_BDD_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>460</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
